Fix test name

Ein-Tim opened the pull request:
## Description
This PR fixes the name of the Cypress test "hotline.cy.js"

## What was changed?
- https://github.com/corona-warn-app/cwa-website/blob/master/cypress/e2e/hotline.cy.js

Thanks to @MikeMcC399 for spotting this in https://github.com/corona-warn-app/cwa-website/pull/3356#issuecomment-1426670908.

MikeMcC399 commented:
Looks better. 
It's a minor point, but now the capitalization of "hotline" is different and inconsistent between the title and the detail.

You might like to add "hotline" to the title of this PR.

![image](https://user-images.githubusercontent.com/66998419/218535506-dce75d23-cb4f-4086-a3b3-230e3572f5b4.png)

Ein-Tim commented:
@MikeMcC399 I changed the title of the PR and fixed the capitalization issue. I squashed the commits and force-pushed to this branch. Hope that's ok with you!

MikeMcC399 reviewed (approved):
Looks 💯 now!

dsarkar commented:
@Ein-Tim @MikeMcC399 Thanks for fixing the cypress test!
